website/packages/docs/src/pages/pkg-webpack-loader.mdx

```diff
@@ -47,67 +47,58 @@ module.exports = {
 };
 ```
 
-### importReact: boolean
+### Options used by `@compiled/webpack-loader`
 
-Will import React into the module if it is not found.
-When using `@babel/preset-react` with the [automatic runtime](https://babeljs.io/docs/en/babel-preset-react#react-automatic-runtime) this is not needed and can be set to `false`.
+`@compiled/webpack-loader` accepts the following options:
 
-Defaults to `true`.
+- `bake`
+- `extensions`
+- `extract`
+- `extractStylesToDirectory`
+- `parserBabelPlugins`
+- `resolve`
+- `ssr`
+- `transformerBabelPlugins`
 
-### importSources: string[]
+#### bake
 
-Additional libraries that should be parsed as though they were `@compiled/react` imports. This is passed directly to `@compiled/babel-plugin`.
+Turns on `@compiled/babel-plugin`.
 
-Specifying this is important if you are using Compiled APIs through another package using the `createStrictAPI` function.
+- Type: `boolean`
+- Default: `true`
 
-### extract: boolean
+#### extensions
 
-Enables extracting all styles to an atomic style sheet,
-to be used in conjunction with `CompiledExtractPlugin`.
-Read the [Webpack CSS extraction](/css-extraction-webpack) guide for help.
+Extensions that we should consider to be code. We use these to identify if a file should be parsed.
 
-Defaults to `false`.
+If set, the value of `extensions` will be used in two places:
 
-### bake: boolean
+1. If `resolver` is not set, then `extensions` will be passed to the default resolver, `enhanced-resolver`.
+2. This is also passed to `@compiled/babel-plugin` under the hood.
 
-Enables baking your styles into JavaScript.
+- Type: `string[]`
+- Default: none, but note that `enhanced-resolver` and `@compiled/babel-plugin` have their own different defaults when the `extensions` option is not set.
```

Comment:
The fact that there isn't a single default being applied everywhere does admittedly sound like a bug

[`enhanced-resolver`](https://github.com/webpack/enhanced-resolve?tab=readme-ov-file#resolver-options) uses `['js', 'json', 'node']` as a default, while `@compiled/babel-plugin` uses `['.js', '.jsx', '.ts', '.tsx']` as a default - I made an issue to track it here https://github.com/atlassian-labs/compiled/issues/1692